morse_test.go

```diff
@@ -17,9 +17,21 @@ func TestEncode(t *testing.T) {
 	}
 	expected := "-. .- -- ."
 	if string(data) != expected {
-		t.Errorf("Expected the value to be %s but was %s", expected, string(data))
+		t.Fatalf("Expected the value to be `%s` but was `%s`", expected, string(data))
+	}
+}
+
+func TestEncodeNotValidLetter(t *testing.T) {
```

Comment:
This is a valid test case. Can you rename this to `TestEncodeValidLetter`

Reply:
Yes, test is valid(Ok). But it is valid test for string with not valid letter(hieroglyph). Could you clarify your opinion?

Reply:
The letters used in this test is not causing an error, we do not have support of any other language other than english yet, therefore the encode should error out in this scenario but currently it is translating partially without informing the user that part of the data was not translated to morse